### ISSUE TITLE
Invalid block scrim hides block options on mobile

### ISSUE THREAD

**Issue by johngodley** [WordPress/gutenberg#9402]
**Describe the bug**
If a block becomes invalid and you are viewing on mobile then the grey scrim hides the block options.

![edit_post_ _wordpress_latest_ _wordpress](https://user-images.githubusercontent.com/1277682/44737384-c096a480-aae9-11e8-9590-4b26af6f4f34.jpg)

**To Reproduce**
1. View editor using a mobile-sized display (i.e. Chrome dev tools > Device)
2. Make a block invalid (remove the trailing `</p>` of a paragraph)
3. Scroll down to the bottom of the block
4. Note that the block dropdown menu, and other buttons, are hidden behind the grey scrim

**Expected behavior**
The dropdown menu button to be available as it is on desktop. I'm not sure about the other buttons

**Comment by johngodley** [comment]
@jasmussen pinging for thoughts. I'm happy to try and fix this, but I'm not sure what it should do - should the scrim stop before the bottom toolbar?

**Comment by jasmussen** [comment]
> should the scrim stop before the bottom toolbar?

Yes, that seems like a good solution.

Could we simply do something like .. and this is pseudo code:

```
.scrim {
	...
	bottom: $block-toolbar-height;

	@include break-small() {
		bottom: 0;
	}
}
```

☝️ this should make the scrim not cover the bottom toolbar, until the break-small breakpoint where if I recall correctly, the block toolbar moves.

**Comment by johngodley** [comment]
Perfect! Used in a fix in #9473
